queries/omen/markets.ts

```diff
@@ -406,6 +408,12 @@ const getMarket = async (params: QueryFixedProductMarketMakerArgs) => {
   return !hasDangerousKeywords ? response : null;
 };
 
+type PrimaryOrderBy =
+  | FixedProductMarketMaker_OrderBy.UsdRunningDailyVolume
+  | FixedProductMarketMaker_OrderBy.CreationTimestamp
```

Comment:
I think we should value more the remaining days, the less the better, then the creation timestamp